### PR TITLE
Add unit to Presto coordinator runtime metrics

### LINKED ISSUE #17770: Set units for the existing runtime metrics
https://github.com/prestodb/presto/pull/17737 added support for setting units runtime metrics. The following changes need to be made:
1. Setting the units for [the existing runtime metrics](https://github.com/prestodb/presto/blob/9ff349f80ef5822b94fa5278e090e73a8fe610c7/presto-common/src/main/java/com/facebook/presto/common/RuntimeMetricName.java)
2. Change the UI logic to make better use of new added information - to display units in the right format.

### PR DESCRIPTION
**Pull request by erick576**
### Add unit to Presto coordinator runtime metrics
- runtime metrics are now associated with their respective unit
- metrics are now evaluated on their unit instead of looking for a substring pattern
- metrics now use the right unit when they weren't checked for before

Resolves #17770 

Test plan - (Please fill in how you tested your changes)

1. Run The Presto Server
2. Visit the UI Page
3. Connect to the CLI
4. Create a table
 > CREATE TABLE ds_customer WITH (format='dwrf', partitioned_by=array['ds']) AS SELECT *, '2021-07-11' as ds FROM customer LIMIT 1000;
5. Insert into table
> INSERT INTO ds_customer SELECT *, '2021-07-12' as ds FROM customer LIMIT 1000;
6. Run a query
> SELECT name, name, name, name, name, name, name FROM ds_customer;
7. View the query details on the UI, and validate the runtime metrics by inspecting and matching the unit with their representation

### Before
Screen Shot 2022-05-18 at 4 08 12 PM

### After
Screen Shot 2022-05-18 at 4 13 12 PM

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx [hunk 1]
```diff
@@ -1073,9 +1073,14 @@ export class QueryDetail extends React.Component {
         }
     }
 
-    renderMetricValue(name, value) {
-      if (name.includes("Nanos")) return formatDuration(parseDuration(value+ "ns"));
-      return formatCount(value);
+    renderMetricValue(unit, value) {
+      if (unit == "NANO") {
```
Comment (zacw7) [inline presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx:1077]: `===` is safer in js.

File: presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx [hunk 2]
```diff
@@ -1073,9 +1073,14 @@ export class QueryDetail extends React.Component {
...
+          return formatDuration(parseDuration(value+ "ns"));
+      } else if (unit == "BYTE") {
+          return formatDataSize(value);
+      } else { // NONE
+          return formatCount(value);
+      }
```
Comment (zacw7) [inline presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx:1083]: nit: `else` are not needed here.

**Review by zacw7 — CHANGES_REQUESTED** [review]
LGTM overall. Thanks for making the change.

File: presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx (hunk 2 above)
Comment (erick576) [reply presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx]: done

**Review by NikhilCollooru — APPROVED** [review]

File: presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx [hunk 3]
```diff
@@ -1073,9 +1073,13 @@ export class QueryDetail extends React.Component {
         }
     }
 
-    renderMetricValue(name, value) {
-      if (name.includes("Nanos")) return formatDuration(parseDuration(value+ "ns"));
-      return formatCount(value);
+    renderMetricValue(unit, value) {
+      if (unit === "NANO") {
+          return formatDuration(parseDuration(value+ "ns"));
+      } else if (unit === "BYTE") {
```
Comment (zacw7) [inline presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx:1079]: `else` is not needed.

Comment (erick576) [reply presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx]: done

**Review by zacw7 — APPROVED** [review]

**Pull request merged**